include/bluetooth/addr.h

```diff
@@ -40,44 +41,89 @@ typedef struct {
 	bt_addr_t a;
 } bt_addr_le_t;
 
+/* Bluetooth device "any" address, not a valid address */
 #define BT_ADDR_ANY     ((bt_addr_t[]) { { { 0, 0, 0, 0, 0, 0 } } })
+/* Bluetooth device "none" address, not a valid address */
 #define BT_ADDR_NONE    ((bt_addr_t[]) { { \
 			 { 0xff, 0xff, 0xff, 0xff, 0xff, 0xff } } })
+/* Bluetooth LE device "any" address, not a valid address */
 #define BT_ADDR_LE_ANY  ((bt_addr_le_t[]) { { 0, { { 0, 0, 0, 0, 0, 0 } } } })
+/* Bluetooth LE device "none" address, not a valid address */
 #define BT_ADDR_LE_NONE ((bt_addr_le_t[]) { { 0, \
 			 { { 0xff, 0xff, 0xff, 0xff, 0xff, 0xff } } } })
 
+/** @brief Compare Bluetooth device addresses.
+ *
+ *  @param a First Bluetooth device address to compare
+ *  @param b Second Bluetooth device address to compare
+ *
+ *  @return negative value if @a a < @a b, 0 if @a a == @a b, else positive
```

Comment:
Hah, this line is so hard to follow with the `@a` references :D (Not wrong, at all, but just fun with all the `@a` interleaved)